playwright.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+  timeout: 30000,
+  globalTimeout: 60000,
+  testDir: "./tests",
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
+  use: {
+    baseURL: "https://traineeautomation.azurewebsites.net/",
+    trace: "on-first-retry",
```

Comment:
You added trace on retry, but your tests are not "retrying" locally, so you would not see any traces locally. Please update it